app/views/idv/doc_auth/agreement.html.erb

```diff
@@ -19,18 +21,25 @@
                        url: url_for,
                        method: 'put',
                        html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
-  <%= check_box_tag(
-        :ial2_consent_given,
-        true,
-        false,
-        class: 'usa-checkbox__input',
-      ) %>
-  <label class="usa-checkbox__label margin-bottom-4" for="ial2_consent_given">
-    <%= t('doc_auth.instructions.consent') %>
-    <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
-  </label>
+  <div class="margin-bottom-4">
+    <%= check_box_tag(
+          :ial2_consent_given,
+          true,
+          false,
+          class: 'usa-checkbox__input',
+          required: true,
+          aria: {invalid: false}
+        ) %>
+    <label class="usa-checkbox__label" for="ial2_consent_given">
+      <%= t('doc_auth.instructions.consent') %>
+      <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
+    </label>
+    <div class="usa-error-message usa-error-message--with-icon display-if-invalid" role="alert">
+      <%= t('sign_up.agree_to_rules_of_use_and_continue') %>
+    </div>
+  </div>
   <%= f.button :button, t('doc_auth.buttons.continue'), type: :submit,
-               class: 'usa-button--big usa-button--wide' %>
+               class: 'usa-button--disabled usa-button--big usa-button--wide' %>
```

Comment:
I remember now we intentionally didn't apply `usa-button--disabled` until JavaScript has run, since otherwise the button always appears disabled in browsers where JavaScript is disabled, even after checking the checkbox ([screenshot](https://user-images.githubusercontent.com/1779930/134177969-dd3dcd00-6ba7-40a6-9f0d-986883450c77.png)).

Was there a reason for adding it, or can we leave it out?

```suggestion
               class: 'usa-button--big usa-button--wide' %>
```